.golangci.yaml

```diff
@@ -297,8 +297,8 @@ linters:
     - spancheck # checks for mistakes with OpenTelemetry/Census spans
     - sqlclosecheck # checks that sql.Rows and sql.Stmt are closed
     - stylecheck # is a replacement for golint
-    - tenv # detects using os.Setenv instead of t.Setenv since Go1.17
```

Comment:
this is deprecated in favour of `usertesting`. Replacing it fixes the warning message we got when running `make golangci-fix`